libcodechecker/server/api/report_server.py

```diff
@@ -2381,11 +2381,10 @@ def massStoreRun(self, name, tag, version, b64zip, force,
                 check_commands, check_durations, cc_version, statistics = \
                     store_handler.metadata_info(metadata_file)
 
-                if len(check_commands) == 0:
-                    command = ' '.join(sys.argv)
-                elif len(check_commands) == 1:
+                command = ''
```

Comment:
Shouldn't we use store a message which indicates that there was no available checker command? By the way how is it possible that metadata.json doesn't contain the checker command? Maybe this was the case in the earlier versions?

Reply:
@bruntib I think if the check command is empty in the database this indicates that no check command was found in the `metadata.json` file. Its not a good idea to store such a message in the database because it can be changed later. Also in the GUI and in the command line we can easily handle this scenario and we can print custom messages. For example now in the UI if there is no check command can be found for a run history entry we don't show the `Check command` link.
I think if someone used earlier version of CodeChecker, we didn't fill this information in the `metadata.json` file